docs/user/troubleshooting.md

```diff
@@ -322,6 +322,37 @@ AQ==
 
 You can then **prepend** that certificate to `client-certificate-authority-data` field in your `${INSTALL_DIR}/auth/kubeconfig`.
 
+### Installer fails to initialize the cluster
```

Comment:
This must be opened against the cluster-ingress-operator to bubble up this error to ClusterOperator object.